pkg/gatherers/clusterconfig/tsdb_status.go

```diff
@@ -0,0 +1,44 @@
+package clusterconfig
+
+import (
+	"context"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"github.com/openshift/insights-operator/pkg/utils/marshal"
+)
+
+// GatherTSDBStatus gathers the Prometheus TSDB status.
+//
+// * Location in archive: config/tsdb/
```

Comment:
Nitpick. The last `/` is not needed AFAIK.

Reply:
Done